lhotse/features/base.py

```diff
@@ -10,7 +10,7 @@
 import torch
 
 from lhotse.audio import Recording
-from lhotse.augmentation import WavAugmenter
+from lhotse.augmentation import AugmentFn, WavAugmenter
```

Comment:
WavAugmenter is no longer useful (?) If it is the case, all appearances of WavAugmenter should be removed within this file

Reply:
Good point. I want to keep it for now so that people with PyTorch older than 1.7 can also use it, but I will probably add a deprecation warning...